lib/api/v1/controllers/auth_test.go

```diff
@@ -0,0 +1,77 @@
+package controllers_test
+
+import (
+	"net/http"
+
+	"github.com/markgravity/golang-ic/lib/api/v1/controllers"
+	"github.com/markgravity/golang-ic/lib/api/v1/forms"
+	"github.com/markgravity/golang-ic/test"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("AuthController", func() {
+	Describe("POST /sign-in", func() {
```

Comment:
You are implementing sign-up but test sign-in

```suggestion
	Describe("POST /sign-in", func() {
```